plugin/gcp/cloudRunServiceLog.go

```diff
@@ -0,0 +1,54 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/puppetlabs/wash/plugin"
+	"google.golang.org/api/logging/v2"
+)
+
+type cloudRunServiceLog struct {
+	plugin.EntryBase
+	*cloudLogFile
+}
+
+func newCloudRunServiceLog(ctx context.Context, apiService cloudRunProjectAPIService, region string, serviceName string) (*cloudRunServiceLog, error) {
+	fields := []cloudLogEntryField{
```

Comment:
I couldn't find the cloud run service equivalent of `gcloud functions log read <function_name>`, so kind of guessed these field values based on symmetry w/ a cloud function's log.

Reply:
https://godoc.org/google.golang.org/api/logging/v2#LogEntry talks about the default when Severity is emptying being `DEFAULT`.

Reply:
Ah thanks, updated.